tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +108,11 @@ def _collect_jars(ctx):
       jars += target.java.transitive_runtime_deps
   return jars
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
```

Comment:
@johnynek @dinowernli Added a comment to the thread, which I think basically implements what @johnynek said above. Would love to hear what you think (preferably in the thread?).